ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/tests/integration/ledger/api/LedgerTestingHelpers.scala

```diff
@@ -159,8 +161,12 @@ class LedgerTestingHelpers(
       verbose: Boolean = false): Future[TransactionTree] = {
     submitAndListenForAllResultsOfCommand(command, transactionFilter, filterCid, verbose).map {
       transactions =>
-        transactions._2 should have length 1
-        transactions._2.headOption.value
+        {
+          withClue("Received transaction tree should have length of 1") {
```

Comment:
Would be useful to see what the transactions look like.

Reply:
Done